gateway/src/apicast/policy/token_introspection/apicast-policy.json

```diff
@@ -20,6 +20,14 @@
       "client_secret": {
         "description": "Client Secret for the Token Introspection Endpoint",
         "type": "string"
+      },
+      "max_ttl_tokens": {
+        "description": "Max TTL for cached tokens",
+        "type": "integer"
```

Comment:
This probably should be > 0  and also have some upper limit.

We should think on how to enforce same limits for everyone on SaaS .

Reply:
I added max and min both for token TTLs and the max tokens to cache. I've chosen some numbers that sounded reasonable to me. Let me know if you agree. It's in the latest commit.